packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -558,7 +612,9 @@ class ScaffoldState extends State<Scaffold> {
 
   @override
   Widget build(BuildContext context) {
-    final EdgeInsets padding = MediaQuery.of(context).padding;
+    EdgeInsets padding = MediaQuery.of(context).padding;
+    if (!config.resizeToAvoidWindowPadding)
+      padding = new EdgeInsets.only(top: padding.top);
```

Comment:
I kind of wanted to use EdgeDims.zero here because the name says to avoid (all) window padding, but things go badly if you skip the top padding here.